### PR TITLE
Add host script/guide.

### PR DESCRIPTION
**Pull request by ericsnekbytes**
In the Lab weekly meeting, participants discussed creating a new script for hosts to follow to ensure fun, effective, high quality weekly meetings. This draft offers a solution. This is based off of [Isabella's Host Script](https://github.com/isabela-pf/jupyter-communitycalls/blob/main/host-guidelines.md) template, and existing work by some JupyterLab community members.

Issue:
- https://github.com/jupyter/docs-team-compass/issues/26

### CODE REVIEW

**Review by JasonWeill — COMMENTED** [review]
Thank you for proposing this guide! Just a few areas that I'm curious about.

File: docs/host_guide.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+The weekly meetings are a way to connect JupyterLab contributors and the
+community, so (ideally) all participants should have a good time, feel welcome,
+and be heard. In line with that:
+
+Meetings should be...
+- Positive, friendly, exciting
+- Welcoming and open to JupterLab users from diverse backgrounds
+- Newcomers in particular should be welcomed and encouraged to
+  chime in and discuss, or participate in whatever way they would like
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst:31]: We can definitely add a suggested way-to-welcome at least. Do you have any suggestions for good ways that don't take up too much time as you noted? And we can also emphasize that people don't have to introduce themselves if they don't want to.

File: docs/host_guide.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Topics
+^^^^^^
+
+Weekly meetings are for contributors to discuss features, ideas, PRs and other
+work on JupyterLab, and for the community to discuss their work, connect
+with JupyterLab contributors, seek help, and more.
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst:38]: Added.

File: docs/host_guide.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+Host Responsibilities
+---------------------
+
+First and foremost, the host should facilitate the flow of a meeting. That means:
+
+- Going through the agenda
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst:45]: Adding now.

File: docs/host_guide.rst [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+- Guiding discussion
+- Making sure everyone has a chance to speak
+- Maintaining order
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst:48]: Adding now.

File: docs/host_guide.rst [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+The host should also:
+
+- Try to log in as the “Project Jupyter” host account during meetings so they
+can manage the meeting (kicking spammers, muting and recording, etc.), more
+info about that HERE [LINK/TODO].
+- Record the meeting. The host starts and stops the recording. These are
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst:57]: Adding now.

File: docs/host_guide.rst [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+with JupyterLab contributors, seek help, and more.
+
+Host Responsibilities
+---------------------
+
+First and foremost, the host should facilitate the flow of a meeting. That means:
+
+- Going through the agenda
+- Guiding discussion
+- Making sure everyone has a chance to speak
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst:48]: Adding now.

File: docs/host_guide.rst [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+The host should also:
+
+- Try to log in as the “Project Jupyter” host account during meetings so they
+can manage the meeting (kicking spammers, muting and recording, etc.), more
+info about that HERE [LINK/TODO].
+- Record the meeting. The host starts and stops the recording. These are
+posted to YouTube for the community to view.
+- Try to keep meeting minutes on the collaborative HackMD document [LINK/TODO]
+
+Last, keep in mind that the code of conduct applies to hosts too!
```
[the remark this answers is not in the document]
Comment (ericsnekbytes) [reply docs/host_guide.rst]: This comes from Isabella's template, I'll tweak this but I think as-written it gets the point across while still keeping the desired tone of this document (ie I think it should strike a positive note for volunteer hosts in the way Isabella's original document does, while being a helpful guide and also not layering on too many requirements so people don't feel they have to meet a thousand different criteria as they're volunteer hosting).

**Review by isabela-pf — APPROVED** [review]

**Comment by fcollonval** [comment]
@ericsnekbytes I see that the link placeholder were left in the document. Would you be able to open a follow-up to add the proper links?

**Comment by ericsnekbytes** [comment]
@fcollonval I forgot to add those, sorry! I'll fix those.

**Comment by ericsnekbytes** [comment]
@fcollonval These have been fixed in https://github.com/jupyterlab/team-compass/pull/226, if you get time I could use a review/approval, thanks!

**Pull request merged**